conanfile.py

```diff
@@ -299,6 +301,9 @@ def source(self):
             self.run("wget -qO- %s.tar.xz | tar -xJ " % url)
         shutil.move("qt-everywhere-src-%s" % self.version, "qt5")
 
+        for patch in glob.glob("patches/*.diff"):
```

Comment:
please rename patch to start with `0001-` and add `sorted(glob.glob(...))` here.
it's important for the future that patches apply in the right and consistent order.